RestCommClient/Classes/RCConnection.m

```diff
@@ -398,6 +403,36 @@ - (void)prepareSounds
         }
         self.callingPlayer.numberOfLoops = -1; // repeat forever
     }
+    
+    // busy
+    filename = @"busy_tone_sample.mp3";
+    // we are assuming the extension will always be the last 3 letters of the filename
+    file = [[NSBundle mainBundle] pathForResource:[filename substringToIndex:[filename length] - 3 - 1]
+                                           ofType:[filename substringFromIndex:[filename length] - 3]];
+    
+    if (file) {
+        self.busyPlayer = [[AVAudioPlayer alloc] initWithContentsOfURL:[NSURL fileURLWithPath:file] error:&error];
```

Comment:
For some reason when I build your changes from Xcode i get a breakpoint here:

![screen shot 2017-09-20 at 2 28 20 pm](https://user-images.githubusercontent.com/8684513/30641892-54ac76d6-9e11-11e7-9c1b-a65d58e37ae9.png)

The only breakpoints I have enabled are exception breakpoints, any idea why that happens?

Reply:
Dont know. I will checkout branch and try it myself and see am I getting the same error.